homeassistant/const.py

```diff
@@ -266,6 +266,7 @@ class Platform(StrEnum):
 LEGACY_CONF_WHITELIST_EXTERNAL_DIRS: Final = "whitelist_external_dirs"
 CONF_XY: Final = "xy"
 CONF_ZONE: Final = "zone"
+DEFAULT_VERIFY_SSL: Final = True
```

Comment:
I have included all the integrations that use `DEFAULT_VERIFY_SSL` as `True`.
I think integrations that use a `False` default can continue using a local constant.